build.gradle

```diff
@@ -1,23 +1,21 @@
-apply from: "$rootDir/tools/jacoco-unified.gradle"
-
 buildscript {
-    ext.kotlin_version = '1.2.71'
-    ext.android_plugin_version = '3.2.0'
+    apply from: "$rootDir/tools/versions.gradle"
+    ext.android_plugin_version = '3.2.1'
     extensions.jacocoVersion = '0.8.2'
```

Comment:
Is it really necessary to declare this? `extensions.jacocoVersion`
What is the difference between this declaration and the one in `versions.gradle`?

Reply:
Because I forgot to remove it! Sorry! hahaha